subsys/net/ip/Kconfig

```diff
@@ -859,6 +859,21 @@ config NET_DEFAULT_IF_WIFI
 
 endchoice
 
+config NET_INTERFACE_NAME
+	bool "Allow setting a name to a network interface"
+	default y
+	help
+	  Allow application to set a name to the network interface in order
+	  to simplify network interface management.
+
+config NET_INTERFACE_NAME_LEN
+	int "Network interface max name length"
+	default 8
```

Comment:
Should we sync with Linux here 1..15 for portability?

Reply:
Sure, we can increase the max size.